src/eslint.config.js

```diff
@@ -0,0 +1,10 @@
+import blumilkDefault from '@blumilksoftware/eslint-config'
+
+export default [
+    ...blumilkDefault,
+    {
+        rules: {
+            'tailwindcss/no-custom-classname': 0,
+        }
+    }
+]
```

Comment:
You don't need this `'tailwindcss/no-custom-classname': 0`, because is the same rule in `blumilkDefault` rules.

```suggestion
export default [
    ...blumilkDefault,
]
```